doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,107 @@
+"""
+===================================================
+Explore and visualize region properties with pandas
+===================================================
+
+This toy example shows how to compute the size of every labelled region in a
+series of 10 images. We use 2D images and then 3D images. The blob-like
+regions are generated synthetically. As the volume fraction (i.e., number of
+pixels or voxels covered by the blobs) increases, the number of blobs
+(regions) decreases, and the size (area or volume) of a single region can get
+larger and larger. The area (size) values are available in a pandas-compatible
+format, which makes for convenient data analysis and visualization.
+
+Beyond area, many other region properties are available.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from skimage import data, measure
+
+
+fractions = np.linspace(0.05, 0.5, 10)
+
+#####################################################################
+# 2D images
+# =========
+
+images = [data.binary_blobs(volume_fraction=f) for f in fractions]
+
+labeled_images = [measure.label(image) for image in images]
+
+properties = ['label', 'area']
+
+tables = [measure.regionprops_table(image, properties=properties)
+          for image in labeled_images]
+tables = [pd.DataFrame(table) for table in tables]
+
+for fraction, table in zip(fractions, tables):
+    table['volume fraction'] = fraction
+
+areas = pd.concat(tables, axis=0)
+
+# Divide the figure into a 2x1 grid
+fig, (im, ax) = plt.subplots(2, 1)
+# Turn off axes for top subplot
+im.axis('off')
+# Plot area vs volume fraction
+areas.plot(x='volume fraction', y='area', kind='scatter', ax=ax)
+# Show image with lowest volume fraction
+ax1 = fig.add_subplot(221)
+ax1.imshow(images[0], cmap='gray_r')
+ax1.set_axis_off()
+ax1.set_title(f'fraction {fractions[0]}')
+# Show image with highest volume fraction
+ax2 = fig.add_subplot(222)
+ax2.imshow(images[-1], cmap='gray_r')
+ax2.set_axis_off()
+ax2.set_title(f'fraction {fractions[-1]}')
+plt.show()
+
+"""
+In the scatterplot, many points seem to be overlapping at low area values.
+To get a better sense of the distribution, we may want to add some 'jitter'
+to the visualization. To this end, we use `stripplot` (from `seaborn`, the
+Python library dedicated to statistical data visualization) with argument
+`jitter=True`.
+"""
+
+sns_fig, sns_ax = plt.subplots()
+sns.stripplot(x='volume fraction', y='area', data=areas, jitter=True,
+              ax=sns_ax)
+# Fix floating point rendering
+sns_ax.set_xticklabels([f'{frac:.2f}' for frac in fractions])
+plt.show()
+
+#####################################################################
+# 3D images
+# =========
+# Doing the same analysis in 3D, we find a much more dramatic behaviour: blobs
+# coalesce into a single, giant piece as the volume fraction crosses ~0.25.
```

Comment:
I still would call it a phase transition, but your wording is totally fine by me. Perhaps @emmanuelle has the definitive answer? Is it normally called a phase transition in this context?

Reply:
I do believe that there is a phase transition here. But to assert it, we would need (at the very least) to do some modelling (e.g., regression), in order to measure (estimate) either the sharp slope change or some power law exponent. So it's not that writing "phase transition" or "completely different behaviour" would be inaccurate -- it may very well be accurate. But for the example to make sense (and be self-contained), if we were to write that, we would need to either derive it, or point to a reference (and not just a general reference, I believe, but one that smoothly connects this toy example to a derivation of some critical volume fraction...).

Reply:
I don't think we need to prove that it's a phase transition. We are not publishing a scientific paper, just illustrating a few scientific ideas. People who need the proof can go read the scientific literature.

Reply:
You can add a sentence here saying "This corresponds to the percolation threshold in statistical physics and graph theory" and link to https://en.wikipedia.org/wiki/Percolation_threshold. No need to say more, most people probably don't care :-).

Reply:
Love it! Done in 73acfbe51a56be0a3e1d687c5090815923890768.